deploy/packages/deb/debian/rules

```diff
@@ -36,8 +36,6 @@ install: build
 	mkdir -p debian/emqx/usr/lib/emqx/bin
 	mkdir -p debian/emqx/usr/lib/emqx/plugins
 	touch debian/emqx/usr/lib/emqx/plugins/.keep
-	mkdir -p debian/emqx/usr/lib/emqx/data/patches
-	touch debian/emqx/usr/lib/emqx/data/patches/.keep
```

Comment:
Q: shouldn't the packages have that directory created to indicate where to put the patches?